examples/javascript/README.md

```diff
@@ -0,0 +1,28 @@
+# An example project of embedding Link SDK within a brand new HTML, CSS and JavaScript application
+
+## Company ID
+You will need to have created a company. 
+
+See <a href="https://github.com/codatio/sdk-link/tree/main#create-a-new-company" target="_blank">readme guide</a> on creating a new company.
+
+## Get started with Vanilla JS
```

Comment:
These docs (including the React example) should point to our official docs rather than recreating here.